CreeDictionary/API/migrations/0003_merge_preverbs.py

```diff
@@ -1,90 +0,0 @@
-# Created by Matt
-
-# The database had two forms of preverbs existing, one with dash, one without
-# e.g. "e" and "ê-" coexist in the database,
-# "e" has definition from MD and "ê-" has definition from CW. They should be merged
-# e.g. "mayi" from MD and "mâyi-" from CW coexist in the database
```

Comment:
this is no longer the case, the source XML file now has only one version

Reply:
Yes, this is one of those things that should be fixed by the linguists, if needed, rather than attempting to sort out with coding.